packages/flutter_tools/test/general.shard/generate_localizations_test.dart

```diff
@@ -2326,4 +2326,49 @@ import 'output-localization-file_en.dart' deferred as output-localization-file_e
     // The original pubspec file should not be overwritten.
     expect(pubspecFile.readAsStringSync(), 'abcd');
   });
+
+  testWithoutContext('can use type: int without specifying a format', () {
+    const String arbFile = '''
+{
+  "orderNumber": "This is order #{number}.",
+  "@orderNumber": {
+    "description": "The title for an order with a given number.",
+    "placeholders": {
+      "number": {
+        "type": "int"
+      }
+    }
+  }
+}''';
+
+    final Directory l10nDirectory = fs.currentDirectory.childDirectory('lib').childDirectory('l10n')
+      ..createSync(recursive: true);
+    l10nDirectory.childFile(defaultTemplateArbFileName)
+        .writeAsStringSync(arbFile);
+
+    try {
```

Comment:
Do you actually need to catch the exception here for the test, or could you let it bubble up to the test runner?

Reply:
I don't think it's needed here, I was just copying from the other tests. Let me just remove this.